app/config/routing.yml

```diff
@@ -19,5 +19,5 @@ overblog_graphql_endpoint:
 
 #overblog_graphql_endpoint:
 #    resource: "@OverblogGraphQLBundle/Resources/config/routing/graphql.yml"
-overblog_graphql_graphiql:
-    resource: "@OverblogGraphQLBundle/Resources/config/routing/graphiql.yml"
\ No newline at end of file
+#overblog_graphql_graphiql:
```

Comment:
Do we really need comments here? We should just remove the code and keep it in the git history if we need it.